ajax/libs/ResponsiveSlides.js/package.json

```diff
@@ -13,5 +13,20 @@
     "type": "git",
     "url": "https://github.com/viljamis/ResponsiveSlides.js"
   },
-  "license": "MIT"
+  "author": "Viljami Salminen <@viljamis> (https://viljamis.com/)",
+  "license": "MIT",
```

Comment:
We should not do the unnecessary position changed, so let's put the `author` information under `license` field. Thank you.